core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java

```diff
@@ -157,11 +157,7 @@ private static boolean initNativeCode() {
      */
     private static boolean loadLibrary() throws IOException {
         String model = System.getProperty("sun.arch.data.model");
```

Comment:
Regarding `String model = System.getProperty("sun.arch.data.model");` - would this be right property to check also for Java OpenJDK (Zulu) distribution?